pkg/services/k8s/apiserver/service.go

```diff
@@ -130,7 +130,7 @@ func (s *service) writeKubeConfiguration(restConfig *rest.Config) error {
 	clusters := make(map[string]*clientcmdapi.Cluster)
 	clusters["default-cluster"] = &clientcmdapi.Cluster{
 		Server:                restConfig.Host,
-		InsecureSkipTLSVerify: restConfig.Insecure,
+		InsecureSkipTLSVerify: true,
```

Comment:
seems like we could disable this if we specify CertificateAuthority or CertificateAuthorityData here using the root ca cert

Reply:
for some reason i can't get the CA data to work, but that seems like an issue unrelated to this PR. @charandas any ideas?